src/shared/find-esp.c

```diff
@@ -238,12 +238,13 @@ static int verify_esp_udev(
 }
 
 static int verify_fsroot_dir(
+                int dir_fd,
+                int fd,
```

Comment:
so `fd` refers to a dir. `dir_fd` also refers to a dir, and says so. kinda confusing.

maybe rename dir_fd → parent_fd? because that's what this is about, no?

Reply:
or just root_fd?